docs/sections/user_guide/cli/tools/mode_template.rst

```diff
@@ -47,14 +47,14 @@ The ``uw`` mode for handling :jinja2:`Jinja2 templates<templates>`.
          Path to file providing override or interpolation values
      --values-format {ini,nml,sh,yaml}
          Values format
+     --env
+         Use environment variables
      --values-needed
          Print report of values needed to render template
      --partial
          Permit partial template rendering
      --dry-run
          Only log info, making no changes
-     --debug
-         Print all log messages, plus any unhandled exception's stack trace (implies --verbose)
```

Comment:
This should have been updated in the PR that removed the `--debug` switch.

Reply:
Oof, thanks for the fix